app-emulation/vmware-modules/vmware-modules-17.5.0-r3.ebuild

```diff
@@ -77,8 +69,16 @@ src_prepare() {
 	default
 }
 
+src_compile() {
+	for mod in ${VMWARE_MODULE_LIST}; do
+		local modlist+=( ${mod}=misc:${S}/${mod}-only )
+	done
+
+	linux-mod-r1_src_compile
+}
+
 src_install() {
-	linux-mod_src_install
+	linux-mod-r1_src_install
 	local udevrules="${T}/60-vmware.rules"
 	cat > "${udevrules}" <<-EOF
 		KERNEL=="vmci",  GROUP="vmware", MODE="660"
```

Comment:
You see the "vmware" group being used here for udev rules? I needs to be created beforehand.

If you're not doing it with the "user" eclass, you need to create a separate group ebuild, as described here: https://devmanual.gentoo.org/ebuild-writing/users-and-groups/

Set ACCT_GROUP_ID to "-1", as described here: https://devmanual.gentoo.org/eclass-reference/acct-group.eclass/index.html

Reply:
There we go, just fixed it! (Sorry about the delay, I made the pr right before going to sleep)